project-koku/koku
# Stop processing on falsey report_period

Fixes [COST-2566](https://issues.redhat.com/browse/COST-2566)

Changes proposed in this PR:
* Stop processing when OCP report period is falsey (provider_uuid and start_date return no report_period data)


Testing Instructions:
1. Set for TRINO processing
2. Execute summary processing for a start_date for a provider_uuid having no report_period data
3. No report period id will be logged and summary processing will not execute

Smoke Test Results:
```
15:04:02 =========================== short test summary info ============================
15:04:02 FAILED tests/rest_api/v1/test_source.py::test_api_ocp_on_aws_source_crud - as...
15:04:02 FAILED tests/rest_api/v1/test_source.py::test_api_ocp_on_azure_source_crud - ...
15:04:02 = 2 failed, 621 passed, 2 skipped, 8385 deselected, 2 warnings in 1338.73s (0:22:18) =
```
[Console Output](https://ci.ext.devshift.net/job/project-koku-koku-pr-check/4839/console)
